component-starter-server/src/main/java/org/talend/sdk/component/starter/server/configuration/ServletConfigurer.java

```diff
@@ -63,4 +62,14 @@ public void doFilter(final ServletRequest servletRequest, final ServletResponse
             filterChain.doFilter(servletRequest, servletResponse);
         }
     }
+
+    public static class EncodingFilter implements Filter {
+
+        @Override
+        public void doFilter(final ServletRequest request, final ServletResponse response, final FilterChain chain) throws IOException, ServletException {
+            request.setCharacterEncoding("UTF-8");
+            response.setCharacterEncoding("UTF-8");
```

Comment:
Is that normal to force the request encoding to UTF8 ? If the client is not UTF8 it will generate an encoding issue no ?